### ISSUE TITLE
73031958 Incomplete registrations associated problems (Eric)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#952]
Priority 10 (0	2	2	6)

Issue reported by **REGManager** at 2017-04-25 06:39:18 UTC
Time: 2017-04-25T06:32:27+00:00
Session ID: 86f17160de3697cae3ce0f3f339b29e7
Problem Page URL: [/userid_details/incomplete_registrations](/userid_details/incomplete_registrations)
Previous Page URL: [https://www.freereg.org.uk/userid_details/options](https://www.freereg.org.uk/userid_details/options)
Reported Issue:
I've been having a look through this report as REGManager and the attached shows the problems.
1) There does not seem to be a sequence.  Name would be best to bring together duplicate entries.
2) Showing the date when registration started would be useful so that we know if it is recent or not.
3) The last in the example shown here is another scam.  It was created on 23 April 2017.  So these are still getting through.
4) There are some duplicate entries - shown here.  The same email address should not be possible.
http://www13.freereg.org.uk/uploads/feedback/screenshot/58feef16f493fd33fe8616e1/Capture.JPG    

![Screenshot](http://www.freereg.org.uk/uploads/feedback/screenshot/58feef16f493fd33fe8616e1/Capture.JPG)

**Comment by Captainkirkdawson** [comment]
It really would have been nice to have received this feedback when did the development checkout of the feature so that items 1 and 2 could have been dealt with during the development phase. Now they will have to be scheduled.
In the meantime please use what is there to make the clean up.
There will always be some "scam" entries until a full trap involving a written response to an image is implemented.

**Comment by edickens** [comment]
This is all part of learning to use a system that we have never had before.  My experience is that it is best to just note these ideas for now because often something better crops up.  We need to get on with V1.5 and then refine the system later.

**Comment by SteveBiggs** [comment]
Another suggestion for future scheduling: having reviewed the list - it would be useful to be able to list incomplete+active registrations, so that those that have been made inactive (because they have some files so can't be destroyed) don't need to be looked at again.

**Comment by edickens** [comment]
And for future scheduling.   Those UserIDs who have no files and have been registered for a year or more.

**Comment by Captainkirkdawson** [comment]
I wonder about the value of making improvements when the current report seems to be unused. There are presently 78 incomplete registrations and none look to be spam or bots

**Comment by edickens** [comment]
I agree that the report does seem to be unused, but I think that there is a problem here.
Who are these people who do not complete registration?  Are they researchers who think that they need to register?  Do we have the registration wording wrong somewhere?  Even many of those who complete registration do not become transcribers.  Jo tries to get people into training, but never hears from some them.  I invite every completed registration to join the transcribers mailgroup, but only about a quarter seem to accept the invitation, so I don't think that they are genuinely looking to transcribe.

**Comment by Captainkirkdawson** [comment]
But those questions are management questions not development features or issues.

**Comment by SteveBiggs** [comment]
I have tried to get SCs to review their lists of incomplete registrations, but like many other requests, unfortunately the majority just don't do it.

**Comment by edickens** [comment]
What should an SC do with an incomplete registration?  I don't think that there is anything that they can do.  They already get an email when someone starts registration so do know of this person.
If these UserIDs are deleted after a period of time (perhaps SysAdmin can do it once a month for year old entries) then this report is needed.  It is also useful to know if there is some other problem should there be an increase.

**Comment by stsccfr** [comment]
Closing as too old to act on now.
